db/migrate/20141231180421_create_reviews.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.string :title, 				null: false
+      t.text :body, 					null: false
+      t.integer :rating, 			null: false
+      t.integer :pizzeria_id, null: false
+      t.integer :user_id, 		null: false
```

Comment:
I would add an index for `user_id` and `pizzeria_id`. This will allow you to quickly look up reviews for a single pizzeria and reviews made by a user.